### ISSUE TITLE
Multiple paths in the browser extension

### ISSUE THREAD

**Issue by taumeister** [buttercup/buttercup-desktop#1106]
Multiple paths in the browser extension.

For example, I have several Windows computers and Linux computers.
On all of them I want to connect my database, which is synchronized via Nextcloud App to the respective computers, so NOT via Webdav but locally.
This means of course that on each device the paths to the same database are different.
But the stupid thing is that the browser addon saves all database paths and synchronizes them.
So on each client, on each browser I have x stored paths, which is totally unnecessary and confusing.

I would have to think of a naming concept for each computer, however, what happens if I have three end devices and 3 different databases ( private, company, friends ), then I have 9 possible databases in my browser extension.

Please reconsider this.

![ksnip_20211201-180836](https://user-images.githubusercontent.com/86101207/144280489-e45642c2-bd6b-4cbd-9c51-a9807947aadb.png)

**Comment by perry-mitchell** [comment]
Good catch.. This is indeed an issue. These should not be sync'd across devices.
